0283_MoveZeroes.py

```diff
@@ -0,0 +1,19 @@
+class Solution(object):
+    def moveZeroes(self, nums):
+        nonzeroi=len(nums)-1
+        i=0
+        while i<nonzeroi:
```

Comment:
This is correct solution, but not an optimal one. What's your complexity for input like [1, 0, 2, 0, 3, 0, 4, 0, 5, 0, 6, 0, 7, 0, 8, 0, 9, 0, 10]? Can you do this in linear time? At this point, you are pushing suffix part but what if you focus on prefix?